rocketmq-console/pom.xml

```diff
@@ -264,6 +269,22 @@
                 <artifactId>findbugs-maven-plugin</artifactId>
                 <version>3.0.4</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.6.2</version>
+                <configuration>
+                    <source>1.8</source>
```

Comment:
You could remove configuration parts while just leave annotationProcessorPaths here. refer to the properties in the pom of RocketMQ.

Reply:
Duplicate maven-compiler-plugin has been removed.